public/js/app/components/ContentHead.jsx

```diff
@@ -68,12 +69,17 @@ const BasicHeaderHead = ({
   onToggleEditor,
   contentIdAndUKey,
 }) => {
+  const store = useStore()
+  const { courseCode } = store
   const { buttons } = i18n.messages[memoLangIndex]
   const { memoInfoByUserLang } = i18n.messages[userLangIndex]
 
+  const language = userLangIndex === 0 ? 'en' : 'sv'
+
   const titleAndInfo = {
     header,
     body: memoInfoByUserLang[contentId].body,
+    link: `${memoInfoByUserLang[contentId].link}${courseCode}?l=${language}` ?? '',
```

Comment:
I was thinking about if we could keep `InfoModal` and `HeadingWithInfoModal` more generic (i.e. revert changes made) and just add the logic up here instead.

The advantages would be that the sub-component will be kept cleaner, but the on the other hand the "ugly" replace part will be here instead.

What du you think?

```suggestion

let { body } = memoInfoByUserLang[contentId]
  if (contentId === 'prerequisites') {
    const language = userLangIndex === 0 ? 'en' : 'sv'
    const editAboutCourseAdminLink = `${SERVICE_URL.aboutCourseAdmin}/edit/${courseCode}?l=${language}`
    body = body.replace('href="LINK_TO_REPLACE"', `href="${editAboutCourseAdminLink}"`)
  }

  const titleAndInfo = {
    header,
    body,
  }
```

Reply:
(have tested my suggestion code, so it's a work in progress to continue with if we deside to change)

Reply:
I refactored this according to your suggestion. I created a function that creates html with a dynamic link inside ContentHead.jsx instead. I also set the language in the url inside the message files.

Reply:
Looks good!